scripts/artifacts-building/containers/build-process.sh

```diff
@@ -123,6 +123,10 @@ ansible_tag_filter "openstack-ansible containers/artifact-build-chroot.yml -e ro
 ansible_tag_filter "openstack-ansible containers/artifact-build-chroot.yml -e role_name=repo_server -v" "install" "config"
 ansible_tag_filter "openstack-ansible containers/artifact-build-chroot.yml -e role_name=rsyslog_server -v" "install" "config"
 
+# test one container build contents
```

Comment:
Well I think the idea is to test if everything can build properly first.

Then we can catch up extra issues there by testing idempotency for a container x. We don't need to test all of these indeed.

Doing it only when push to mirror doesn't bring anything, it only reduces our coverage of testing. I'd be enclined to put all the tests before the ``PUSH_TO_MIRROR`` but some rely on pushes to be done (see below), because they are testing the upload.